api/internal/user/serializers.py

```diff
@@ -14,6 +14,8 @@ class Meta:
             "name",
             "external_id",
             "owners",
+            "terms_agreement",
+            "terms_agreement_at",
```

Comment:
I believe we don't need to add these here as we're consuming them via the graphql resolvers. Or did you have another reason to add them here for?

Reply:
This is requested for a change in the `internal/user` endpoint as part of being able to login without a service provider.
https://github.com/codecov/engineering-team/issues/390#issue-1871820224
The current graphql endpoints are dependent on a service provider, however another ticket is to add graphql queries to be independent of services.

Reply:
A gotcha, was not aware, ty for that!